### PR TITLE
small fixes and enhancements to new overlays

### LINKED ISSUE #4757: Metadata on new extended and mouse hover option no more visible after scrolling and come back to a thumbnail
**Describe the bug**
Metadata text on bottom thumbnail area disappear on last new mode after scrolling back and then come back on image. @AlicVB 

**To Reproduce**
1. On new star icon popover menu, select the last one (always extended and mouse hover)
2. Move mouse to an image (could also be made on other ones) and see extended metadata text visible as it should on the thumbnail
3. Then scroll down to make the images where metadata text were shown disappear
4. Then scoll up to come back to the images viewed on step 2 and move mouse over same images. See that metadata text is no more visible (but extended background is still there).

**Expected behavior**
Of course, having metadata still there each time we move the mouse hover it.

**Screenshots**
Below, a screenshot of the issue on step 4:
Capture-20200418160044-1922x1054

**Platform (please complete the following information):**
 - Darktable Version: last master
 - OS: Ubuntu 20.04

### LINKED ISSUE #4758: Add a tooltip to star icon as a reminder of the new possibility for new users
@AlicVB: 
Just a simple and missing one on the new star icon popover menu: a tooltip when moving on star icon explain quickly what is possible now with this icon.
Tooltip is actually missing and having one could be helpful for new users and even old ones who will need to discover new features.

### PR DESCRIPTION
**Pull request by AlicVB**
fix #4757, fix #4758

This content : 
- fix `dt_thumbnail_???` css class that wasn't set correctly
- add a tooltip to the overlays icon
- fix the extended content with overlays mixed and new thumbs
- fix extended text height if content is different from 2 lines
- add 2 new variables for patterns : `$(RATING_ICONS)` to show reject/stars icons with utf8 chars like in the filter dropdown and `$(SIDECAR_TXT)` to display the content of a sidecar txt file if any. This last one as been added by default to the "extended" pattern, knowing that nothing will be written if no sidecar is present. This replace the specific overlay used actually.

### CODE REVIEW

**Comment by Nilvus** [comment]
Good job, both issues are indeed fixed. Thanks.

File: data/darktableconfig.xml.in [hunk 1]
```diff
@@ -721,7 +721,7 @@
   <dtconfig prefs="gui" section="lighttable">
     <name>plugins/lighttable/extended_pattern</name>
     <type>string</type>
-    <default>$(FILE_NAME).$(FILE_EXTENSION)$(NL)$(EXIF_EXPOSURE) f/$(EXIF_APERTURE) $(EXIF_FOCAL_LENGTH)mm ISO $(EXIF_ISO)</default>
+    <default>$(FILE_NAME).$(FILE_EXTENSION)$(NL)$(EXIF_EXPOSURE) f/$(EXIF_APERTURE) $(EXIF_FOCAL_LENGTH)mm ISO $(EXIF_ISO)$(SIDECAR_TXT)</default>
```
Comment (TurboGit) [inline data/darktableconfig.xml.in:724]: Using this to configure the txt is better indeed, but in that case I suppose we want to remove the old prefs which was used to activate or not the txt.

Comment (AlicVB) [reply data/darktableconfig.xml.in:724]: no, this is still used in culling preview...

**Comment by TurboGit** [comment]
So sounds good to me now. Really nice job as usual. Thanks.

**Pull request merged**